tests/track/test_tracker_manager.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+
+import mlrun.track
+from mlrun.track import BaseTracker, Tracker
+from mlrun.track.tracker_manager import TrackerManager
+from mlrun.track.trackers.mlflow_tracker import MLFlowTracker
+
+
+class TestTracker(Tracker):
+    # just some random module
+    TRACKED_MODULE_NAME = "os"
+
+    def pre_run(self, context) -> dict:
+        return context.to_dict()
+
+    def post_run(self, context):
+        return True
+
+
+class TestBaseTracker(BaseTracker):
```

Comment:
`pytest` will try to pick tests methods out of classes that start with the word Test. Not a good practice to do so.

```suggestion
class BaseTrackerExample(BaseTracker):
```